### PR TITLE
Pocket platform pages

### PR DESCRIPTION
**Pull request by reemhamz**
## Description
Migrating over some Pocket pages. This PR is for the platform pages that have the exact same styling, but I'm using Jinja's template inheritance to code them. 

## Issue / Bugzilla link
#10211 

## Testing
- [Android page](http://localhost:8000/en-US/external/pocket/android/)
- [iOS page](http://localhost:8000/en-US/external/pocket/ios/)
- [Safari page](http://localhost:8000/en-US/external/pocket/safari/)
- [Opera page](http://localhost:8000/en-US/external/pocket/opera/)
- [Chrome page](http://localhost:8000/en-US/external/pocket/chrome/)
- [Edge page](http://localhost:8000/en-US/external/pocket/edgei/)

### CODE REVIEW

File: bedrock/externalpages/templates/externalpages/pocket/android.html [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
 
-{% block title %} Android {% endblock%}
+{% extends "externalpages/pocket/platform-pages.html" %}
 
-{% block content %}
+{% set platform_image = 'img/externalpages/pocket/welcome-android.jpg' %}
```
Comment (nathan-barrett) [inline bedrock/externalpages/templates/externalpages/pocket/android.html:9]: Killer! I'm glad you found a way to make these work :)

File: bedrock/externalpages/templates/externalpages/pocket/platform-pages.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+{% block page_title %} Add Pocket to Chrome {% endblock %}
+
+{% block pocket_css %}
+ {{ css_bundle('pocket-platforms') }}
+{% endblock %}
+
+{% block content %}
+  <div class="platform">
+    <div class="platform__content">
```
Comment (nathan-barrett) [inline bedrock/externalpages/templates/externalpages/pocket/platform-pages.html:19]: Wondering if we're not using the `.platform__content` class if we need an extra container here?

**Review by nathan-barrett — APPROVED** [review]
r+wc 🥮 Nice job! just a few *nits* - and wondering if we need to switch any colors back to the pocket theme hex - otherwise looks good!

File: bedrock/externalpages/templates/externalpages/pocket/platform-pages.html (hunk 2 above)
Comment (reemhamz) [reply bedrock/externalpages/templates/externalpages/pocket/platform-pages.html:19]: good eye!

**Pull request merged**